filterpy/kalman/sigma_points.py

```diff
@@ -500,13 +500,14 @@ def sigma_points(self, x, P):
 
         lambda_ = n / (n + 1)
         Istar = np.array([[-1/np.sqrt(2*lambda_), 1/np.sqrt(2*lambda_)]])
+
         for d in range(2, n+1):
             row = np.ones((1, Istar.shape[1] + 1)) * 1. / np.sqrt(lambda_*d*(d + 1))
             row[0, -1] = -d / np.sqrt(lambda_ * d * (d + 1))
             Istar = np.r_[np.c_[Istar, np.zeros((Istar.shape[0]))], row]
 
         I = np.sqrt(n)*Istar
-        scaled_unitary = U.dot(I)
+        scaled_unitary = (U.T).dot(I)
```

Comment:
Actually U.T.dot(I) would be enough (https://docs.python.org/3/reference/expressions.html#grammar-token-attributeref). Anyway it's only a matter of style.